app/models/declaration/ExportItem.scala

```diff
@@ -45,11 +45,10 @@ case class ExportItem(
   val isCompleted: PartialFunction[DeclarationType, Boolean] = {
     case DeclarationType.STANDARD | DeclarationType.SUPPLEMENTARY =>
       procedureCodes.isDefined && isFiscalInformationCompleted && statisticalValue.isDefined &&
-        packageInformation.nonEmpty && commodityMeasure.isDefined && additionalInformation.isDefined
+        packageInformation.nonEmpty && commodityMeasure.isDefined
```

Comment:
So additionalInformation is not required for ANY dec type.  Is that correct?

Reply:
Yes, confirmed with @bethany-kish